.github/workflows/main_and_pr_workflow.yml

```diff
@@ -8,22 +8,28 @@ jobs:
   build:
     strategy:
       matrix:
-        java-version: [ 11, 17 ]
+        entry:
+          - { opensearch-version: 1.3.6, java-version: 11 }
+          - { opensearch-version: 1.3.6, java-version: 17 }
+          - { opensearch-version: 1.3.6, java-version: 19 }
+          - { opensearch-version: 2.3.0, java-version: 11 }
+          - { opensearch-version: 2.3.0, java-version: 17 }
+          - { opensearch-version: 2.3.0, java-version: 19 }
         runs-on: [ubuntu-latest, macos-latest, windows-latest]
-    name: Build on ${{ matrix.runs-on }} with jdk ${{ matrix.java-version }}
+    name: Build on ${{ matrix.runs-on }} with jdk ${{ matrix.entry.java-version }} against OpenSearch ${{ matrix.entry.opensearch-version }}
```

Comment:
```suggestion
    name:  ${{ matrix.runs-on }}, jdk ${{ matrix.entry.java-version }}, OpenSearch ${{ matrix.entry.opensearch-version }}
```
Make name shorter otherwise it's impossible to see difference in dropdown,,,
![image](https://user-images.githubusercontent.com/403174/200640078-d0bdb6a6-04e2-4de4-9196-4c5fc91858ec.png)